indico/web/client/styles/base/_typography.scss

```diff
@@ -55,6 +55,181 @@
   font-size: 13px;
 }
 
-@mixin text-superfluous {
+html,
```

Comment:
I think something went wrong here. I split this file and moved all the rules that generate CSS rules (ie everything except for the mixins) to `_typography_elements.scss` so `_typography.css` only contains mixins.

Reply:
I'll just overwrite this with your version, and hope I didn't make any changes to this one. :)

Reply:
that should be fine. in https://github.com/indico/indico/commit/52a7363a5f3496a135fcab86acb9406f91ed2330 I just split it without making any other changes